jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TableFilteringSection.java

```diff
@@ -0,0 +1,14 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.elements.complex.table.Table;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JTable;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+
+public class TableFilteringSection {
+    @UI("#mat-input-84")
```

Comment:
This identifier is automatically generated by the angular framework. Is another locator possible here?